forage-android/src/main/java/com/joinforage/forage/android/core/element/state/PanElementStateManager.kt

```diff
@@ -90,6 +92,12 @@ internal class PanElementStateManager(state: ElementState, private val validator
             .firstOrNull { it != null }
     }
 
+    private fun inferCardInfo(cardNumber: String): DerivedCardInfo {
+        return DerivedCardInfo(
```

Comment:
This is great! Love that we're returning a new instance of `DerivedCardInfo` each time instead of mutating an existing instance. Nice!!